wordpress-mobile/WordPress-iOS
# Blaze: Display overlay only once

Closes #20251

## Description
This PR changes the blaze overlay logic to be only shown once per entry point. After that, the blaze web flow is presented directly.

## Testing Instructions

1. Switch to a Blaze-eligible site
2. Switch to the "Home" tab
3. Tap on the Blaze dashboard card
4. ✅ A Blaze overlay should be displayed
5. Dismiss the overlay
6. Tap on the Blaze dashboard card again
7. ✅ A Blaze webview opens
8. Repeat the above steps for the menu item, posts list, and pages list entry points

## Regression Notes
1. Potential unintended areas of impact
Features removal overlays frequency

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Testing manually and depended on `OverlayFrequencyTrackerTests`

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
